rmc/utils/mpc.py

```diff
@@ -688,6 +749,19 @@ def annotate_mpc(
         hl.tstr
     ), "'locus' must be a LocusExpression, and 'alleles' must be an array of strings!"
 
-    mpc_ht = mpc_release.ht()
-    ht = ht.annotate(mpc=mpc_ht[ht.key].mpc)
+    if "transcript" not in ht.row or add_transcript_annotation:
```

Comment:
Just a note for later that this does not allow annotation with all MPC values for duplicate variants, in case that is something we want to do in the future.